testify/test_rerunner.py

```diff
@@ -1,11 +1,14 @@
 from __future__ import absolute_import
 from itertools import groupby
 import sys
+from collections import namedtuple
 
 from . import test_discovery
 from .test_runner import TestRunner
 
 
+Test = namedtuple('Test', 'module cls, method')
```

Comment:
mixing spaces, commas and stringly typed :)  I'd prefer `namedtuple('Test', ('module', 'cls', 'method'))`